setup.cfg

```diff
@@ -19,8 +19,15 @@ classifiers =
 packages = find:
 python_requires = >=3.7
 install_requires =
-    deeplabcut>=2.2.0.2
     ndx-pose
-
+    pynwb
+    numpy
+    pandas
+    opencv-python
+    hdmf
```

Comment:
I think pandas expects the `tables` package rather than `hdmf` to load/write h5 files. Could you please confirm?

Reply:
Pandas has `tables` as a dependency already:
https://github.com/pandas-dev/pandas/blob/1ab02e474edd7542a59ddbd3f9104dd94a7aec23/requirements-dev.txt#L35

The hdfm dependency here is for some specialized IO warnings here:
https://github.com/DeepLabCut/DLC2NWB/blob/e4d4bca9b1a8842365d308a24c7c15c1abb1b704/dlc2nwb/utils.py#L211

Reply:
The pandas requirements.txt is generated from this one: https://github.com/pandas-dev/pandas/blob/1ab02e474edd7542a59ddbd3f9104dd94a7aec23/environment.yml, where `tables` is optional. I've just checked on a fresh environment: pandas does complain when writing to hdf. Let's just add it to the list, and I can merge the PR.

Reply:
Thanks for the clarification!